src/BugsnagUnity/Native/Android/NativeInterface.cs

```diff
@@ -9,6 +9,27 @@
 
 namespace BugsnagUnity
 {
+    internal class DisposableContainer
```

Comment:
Any reason not to make `DisposableContainer` implement `IDisposable` as well?

Reply:
done 👍